### PR TITLE
[feat][fix] Logout and fixes

### PR DESCRIPTION
**Pull request by shivampandey0**
- Logout button on wishlist and cart pages
- added more products
- Out of stock product added
- Favicons added
- CTA color change on the wishlist page
- Price range Slider from component library
- Issues linked with CL components.

image

### CODE REVIEW

**Review by ZuberDunge — APPROVED** [review]
Excellent code structure, Everything looks good to me. Good to go from my side.

File: index.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Favicons -->
+    <link rel="icon" type="image/png" href="./assets/favicon.ico" />
```
Comment (nareshpoonia) [inline index.html:8]: I think you can move this favicon link to your main css, that way you wouldn't have to link it in all of your html

**Review by nareshpoonia — APPROVED** [review]
Code looks clean, just one comment on favicon icon.
Good to merge

File: index.html (hunk 1 above)
Comment (shivampandey0) [reply index.html:8]: The favicon can be linked only in the header.
Thanks for the review

**Pull request merged**